app/assets/stylesheets/templatemo-style.scss

```diff
@@ -0,0 +1,403 @@
+    background: #28a7e9;
+    border-radius: 50%;
+    color: #fff;
+    display: inline-block;
+    width: 40px;
+    height: 40px;
+    line-height: 40px;
+    text-align: center;
+    margin-right: 20px;
+  }
+}
+/* end feature1 */
+/* start pricing */
+
+#pricing {
+    background: #f8f8f8;
+    text-align: center;
+    padding-top: 80px;
+    padding-bottom: 80px;
+}
+#pricing h2 {
+    padding-bottom: 60px;
+}
+#pricing .pricing {
+    background: #fff;
+}
+#pricing .active {
+    position: relative;
+    bottom: 20px;
+}
+#pricing .pricing .pricing-title {
+    background: #28a7e9;
+    color: #fff;
+    font-weight: bold;
+    padding: 30px;
+}
+#pricing .pricing .pricing-title p {
+    font-size: 20px;
+}
+#pricing .pricing ul {
+    padding: 0;
+    margin: 0;
+}
+#pricing .pricing ul li {
+    display: block;
+    list-style: none;
+    padding: 16px;
+}
+#pricing .pricing .btn {
+    background: transparent;
+    border: 1px solid #28a7e9;
+    border-radius: 0px;
+    color: #28a7e9;
+    font-weight: bold;
+    padding-right: 40px;
+    padding-left: 40px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+    transition: all 0.3s ease;
+}
+#pricing .pricing .btn:hover {
+    background: #28a7e9;
+    color: #fff;
+}
+/* end pricing */
+/* start download */
+
+#download {
+    padding-top: 80px;
+    padding-bottom: 80px;
+}
+#download h2 {
+    padding-bottom: 20px;
+}
+#download .btn {
+    border-radius: 0px;
+    font-weight: bold;
+    margin-top: 20px;
+    padding: 10px 40px 10px 40px;
+}
+/* end download */
+/* start contact */
+
+#contact {
+    background: image_url('contato-bg.jpg') no-repeat;
+    background-size: cover;
+    color: #fff;
+}
+#contact h2 {
+    padding-bottom: 10px;
+}
+#contact address {
+    padding-top: 20px;
+}
+#contact address .fa {
+    background: #28a7e9;
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    line-height: 40px;
+    text-align: center;
+    margin-top: 6px;
+    margin-right: 10px;
+}
+#contact .contact-form {
+    padding-top: 40px;
+}
+#contact .form-control {
+    border: none;
+    border-radius: 0px;
+    box-shadow: none;
+    margin-bottom: 20px;
+}
+#contact input {
+    height: 50px;
+}
+#contact input[type="submit"] {
+    background: #28a7e9;
+    color: #fff;
+    font-weight: bold;
+    transition: all 0.3s ease;
+}
+#contact input[type="submit"]:hover {
+    background: #107DB6;
+}
+/* end contact */
+/* start footer */
+
+footer {
+    background: #fff;
+    font-weight: 400;
+    text-align: center;
+    padding: 20px;
+}
+/* end footer */
+
+@media screen and ( max-width: 991px) {
+    .templatemo-box {
+        margin-bottom: 30px;
+    }
+    #pricing .active {
+        bottom: 0;
+    }
+    .pricing {
+        margin-bottom: 20px;
+    }
+    #feature img,
+    #download img {
+        margin-top: 30px;
+    }
+    #feature1 img {
+        margin-bottom: 30px;
+    }
+}
+@media screen and ( max-width: 767px) {
+    .navbar-default .nav li a {

```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector should have depth of applicability no greater than 2, but was 4